src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,106 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
 import './FindMovie.scss';
 
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ addMovie }) => {
+  const [value, setSearchTerm] = useState('');
+  const [results, setSearchResults] = useState({
+    title: '',
+    description: '',
+    imgUrl: '',
+    imdbUrl: '',
+    imdbId: '',
+  });
+  const [filmNotFound, setNotFound] = useState(false);
+
+  const handleChange = (event) => {
+    event.preventDefault();
```

Comment:
```suggestion
```
`preventDefault` is only needed to prevent the common behavior on form submitting. For input changes it is redundant and does nothing